core/src/test/java/io/jenkins/pluginhealth/scoring/probes/ThirdPartyRepositoryDetectionProbeTest.java

```diff
@@ -26,19 +25,31 @@ ThirdPartyRepositoryDetectionProbe getSpy() {
         return spy(ThirdPartyRepositoryDetectionProbe.class);
     }
 
-    static Stream<Arguments> pomPathAndProbeResults() {
+    static Stream<Arguments> pomPathAndProbeSuccessTestParameters() {
+        return Stream.of(
+            arguments(
+                Paths.get("src","test","resources","pom-test-only-correct-path"),
+                new String[]{SCMLinkValidationProbe.KEY},
+                Map.of(SCMLinkValidationProbe.KEY, ProbeResult.success(SCMLinkValidationProbe.KEY, ""))
+            ),
+            arguments(
+                Paths.get("src","test","resources","pom-test-no-repository-tag"),
+                new String[]{SCMLinkValidationProbe.KEY},
+                Map.of(SCMLinkValidationProbe.KEY, ProbeResult.success(SCMLinkValidationProbe.KEY, ""))
+            )
+        );
+    }
+    static Stream<Arguments> pomPathAndProbeFailureTestParameters() {
         return Stream.of(
             arguments(
                 Paths.get("src","test","resources","pom-test-both-paths"),
-                "test-foo-bar",
                 new String[]{SCMLinkValidationProbe.KEY},
-                Map.of(ThirdPartyRepositoryDetectionProbe.KEY, ProbeResult.failure(ThirdPartyRepositoryDetectionProbe.KEY, "Third party repositories detected in the plugin"))
```

Comment:
Though this change passed the test case. I am not sure if it is correct.

Because I believe `ProbeResult.failure(ThirdPartyRepositoryDetectionProbe.KEY, "Third party repositories detected in the plugin")` should stay.